tensorboard/plugins/image/tf_image_dashboard/tf-image-loader.html

```diff
@@ -58,31 +58,24 @@
       </template>
     </tf-card-heading>
     <div id="main-image-container"></div>
-    <div style="display: flex; flex-direction: row;">
-      <paper-icon-button
-        selected$="[[_expanded]]"
-        icon="fullscreen"
-        on-tap="_toggleExpanded"
-      ></paper-icon-button>
-      <paper-icon-button
-        icon="aspect-ratio"
-        on-tap="_showActualSize"
-        title="Show the image at its true pixel size"
-      ></paper-icon-button>
-    </div>
 
     <style>
       :host {
         display: block;
-        width: 330px;
+        width: 332px;
```

Comment:
Why? I know that you added a 2px-total border to the image container, but you removed one from the image, so it should be par for the course, right? (It's also not necessary to try to preserve the absolute dimensions of all UI elements when changing other things. There's a reason that we don't do screendiff testing.)